src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/servicebus/ServiceBusHelper.java

```diff
@@ -69,7 +75,17 @@ Message mapToBusMessage(Msg msg) {
         busMessage.setMessageId(msg.getMsgId());
         busMessage.setMessageBody(getMsgBodyInBytes(msg));
         busMessage.setLabel(msg.getLabel());
-
+        busMessage.setProperties(new HashMap<String,String>());
+        RequestTelemetryContext context = ThreadContext.getRequestTelemetryContext();
```

Comment:
when we start assigning request telemetry `getHttpRequestTelemetry ` should not be null